containers/centos-9.containerfile

```diff
@@ -14,6 +14,7 @@ RUN echo v1 \
         gcc \
         git \
         libguestfs-tools-c \
+        libxml2-devel \
```

Comment:
Why is this needed? maybe add a comment?

Reply:
Dropped